vars/pipeline2ATX.txt

```diff
@@ -5,6 +5,9 @@ The report is compressed as a zip file within the build workspace and can be upl
 <ul>
 <li><a href="https://plugins.jenkins.io/pipeline-utility-steps/">Pipeline Utility Steps Plugin</a></li>
 <li><a href="https://plugins.jenkins.io/pipeline-stage-view/">Pipeline Stage View Plugin</a></li>
+<li>Environment variable "TEST_LEVEL" with a string of the corresponding test level</li>
+<li>Environment variable "PRODUCT_NAME" with the name of the software product</li>
+<li>Environment variable "PRODUCT_VERSION" with the version number of the software product</li>
```

Comment:
Those env variables are no prerequisites. The step works fine even someone missed to define them. I rather would document them separately because it is just an offer to structure all pipelines using TEST-GUIDE coverages.